device-types/Albis-Elcon/BIG2862.yaml

```diff
@@ -0,0 +1,38 @@
+---
+manufacturer: Albis-Elcon
+model: BIG2862
+slug: big2862
```

Comment:
please append `albis-elcon-` to the slug

Reply:
done